packages/amazon-bedrock/CHANGELOG.md

```diff
@@ -0,0 +1 @@
+# @ai-sdk/amazon-bedrock
```

Comment:
file can be removed, will be automatically created by changesets

Reply:
removed